www/docs/src/content/docs/how-it-works/index.mdx

```diff
@@ -0,0 +1,77 @@
+- **`virtual:studiocms/version`**: The Virtual module that provides the current version of the StudioCMS integration ecosystem.
+- **`virtual:studiocms/_nav`**: The Virtual module that provides the navigation mappings for the userfacing Front-end.
+- **`virtual:studiocms/astromdremarkConfig`**: The Virtual module that provides the configuration for the Astro markdown support.
+- **`studiocms:components`**: The Virtual module that provides the front-end exported components for StudioCMS.
+- **`studiocms:helpers`**: The Virtual module that provides the front-end exported helper functions for StudioCMS.
+
+### StudioCMS: Dashboard
+
+#### Overview
+
+The StudioCMS Dashboard is a web-based interface that allows you to manage your AstroStudioCMS project. It provides a user-friendly interface for creating, editing, and deleting content for your Astro project. The StudioCMS Dashboard is built with [Astro](https://astro.build) and [DaisyUI](https://daisyui.com/) with [UnoCSS](https://unocss.dev).
+
+#### Features
+
+- **Dashboard**: The StudioCMS Dashboard provides a user-friendly interface for managing your AstroStudioCMS project.
+- **UnoCSS**: The StudioCMS Dashboard uses UnoCSS for styling and theming.
+- **DaisyUI**: The StudioCMS Dashboard uses DaisyUI for styling and theming.
+- **AuthConfig**: The StudioCMS Dashboard provides authentication configuration options.
+- **Astrolace**: The StudioCMS Dashboard is built using [Astrolace](https://github.com/matthiesenxyz/astrolace) a [Shoelace.style](https://shoelace.style) Astro Integration.
+- **DashboardRouteOverride**: The StudioCMS Dashboard provides a dashboard route override configuration option. Allowing you to change the default route for the StudioCMS Dashboard and API.
+
+#### Dashboard Virtual Modules
+
+- **`studiocms-dashboard:auth`**: The Virtual module that provides the authentication configuration for the StudioCMS Dashboard.
+- **`studiocms-dashboard:components`**: The Virtual module that provides the exported dashboard components for the StudioCMS Dashboard.
+- **`studiocms-dashboard:contentHelpers`**: The Virtual module that provides the exported content helper functions for the StudioCMS Dashboard.
+- **`studiocms-dashboard:routeMap`**: The Virtual module that provides the exported routes for the StudioCMS Dashboard.
+
+### StudioCMS: ImageHandler
+
+#### Overview
+
+The StudioCMS ImageHandler is an integration that provides a custom image component for your Astro project. This component is used to handle remote images and optimize them for your Astro project in an SSR environment using [@unpic/astro](https://unpic.pics/img/astro/) for image optimization.
+
+#### Features
+
+- **`<CustomImage />`**: The StudioCMS ImageHandler provides a custom image component for your Astro project.
+
+#### ImageHandler Virtual Modules
+
+- **`studiocms:imageHandler/components`**: The Virtual module that provides the exported image components for the StudioCMS ImageHandler.
```

Comment:
Why isn't this one `virtual://////...`

Reply:
`virtual:...` vs `studiocms:...`  due to the naming convention of [Vite](https://vitejs.dev/guide/api-plugin#conventions) setting the rules between "PUBLIC" and "internal" differences.  the `studiocms` virtual modules also have a .dts for the user, while the `virtual` ones do not (since they are internal use(But could be accessed still without typings, this is useful for someone looking to customize the entire project as the "How it works" section is meant as a technical breakdown this can be considered the "Advanced section"))